src/Pages/Books.js

```diff
@@ -0,0 +1,11 @@
+import AddBook from '../Components/AddBook';
+import BookList from '../Components/bookList';
+
+export default function Books() {
+  return (
+    <div>
+      <BookList />
+      <AddBook />
```

Comment:
- [ ] `AddBook` component should be imported and used inside `BookList` component